doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -14,101 +19,237 @@ For testing purposes only. For testing purposes only. For testing purposes only.
 
 ## Prerequisites
 
+### Terraform
```

Comment:
I can't comment on the line direction, but I think the "testing purposes only" above can go, now that `terraform-disclaimer` exists

Reply:
Updated in patch